apps/web/pages/settings/platform/index.tsx

```diff
@@ -0,0 +1,237 @@
+                    OAuth Clients
+                  </h1>
+                  <p className="text-default text-sm ltr:mr-4 rtl:ml-4">
+                    Connect your platform to cal.com with OAuth
+                  </p>
+                </div>
+                <div>
+                  <NewOAuthClientButton />
+                </div>
+              </div>
+              {Array.isArray(data) && data.length ? (
+                <>
+                  <div className="border-subtle rounded-b-lg border border-t-0">
+                    {data.map((client, index) => {
+                      return (
+                        <OAuthClientCard
+                          name={client.name}
+                          redirectUris={client.redirectUris}
+                          bookingRedirectUri={client.bookingRedirectUri}
+                          bookingRescheduleRedirectUri={client.bookingRescheduleRedirectUri}
+                          bookingCancelRedirectUri={client.bookingCancelRedirectUri}
+                          permissions={client.permissions}
+                          key={index}
+                          lastItem={data.length === index + 1}
+                          id={client.id}
+                          secret={client.secret}
+                          isLoading={isDeleting}
+                          onDelete={handleDelete}
+                          areEmailsEnabled={client.areEmailsEnabled}
+                        />
+                      );
+                    })}
+                  </div>
+                </>
+              ) : (
+                <EmptyScreen
+                  headline="Create your first OAuth client"
+                  description="OAuth clients facilitate access to Cal.com on behalf of users"
+                  Icon="plus"
+                  className=""
+                  buttonRaw={<NewOAuthClientButton />}
+                />
+              )}
+            </div>
+            <div>
+              <div className="border-subtle mx-auto block justify-between rounded-t-lg border px-4 py-6 sm:flex sm:px-6">
+                <div className="flex w-full flex-col">
+                  <h1 className="font-cal text-emphasis mb-1 text-xl font-semibold leading-5 tracking-wide">
+                    Managed Users
+                  </h1>
+                  <p className="text-default text-sm ltr:mr-4 rtl:ml-4">
+                    See all the managed users created by your OAuth client.
+                  </p>
+                </div>
+                {Array.isArray(data) && data.length && (
+                  <div>
+                    <Dropdown modal={false}>
+                      <DropdownMenuTrigger asChild>
+                        <Button color="secondary">{initialClientName}</Button>
+                      </DropdownMenuTrigger>
+                      <DropdownMenuContent>
+                        {data.map((user) => {
```

Comment:
user ? the data is containing oAuthCLients no ?

Reply:
The data here contains managed users of a particular oauth client, thats why the user

Reply:
the data here seems to be an array of OAuthClients, isn't this a dropdown select to choose which oauth clients you want to see the managed users of ?

Reply:
Ah yes right, my bad sorry.